platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongListHeap.java

```diff
@@ -75,16 +76,25 @@ public LongListHeap(final int numLongsPerChunk) {
     /**
      * Create a {@link LongListHeap} from a file that was saved.
      *
+     * @param file the file to read from
      * @throws IOException If there was a problem reading the file
      */
     public LongListHeap(final Path file) throws IOException {
         super(file, DEFAULT_RESERVED_BUFFER_LENGTH);
     }
 
-    public LongListHeap(final Path file, final long reservedBufferLength) throws IOException {
+    /**
+     * Create a {@link LongListHeap} from a file that was saved.
+     *
+     * @param file the file to read from
+     * @param reservedBufferLength the number of bytes to reserve for the buffer
+     * @throws IOException If there was a problem reading the file
+     */
+    LongListHeap(final Path file, final long reservedBufferLength) throws IOException {
```

Comment:
What's the use case for this? Should `reservedBufferLength` be always 0 for LongListHeap?

Reply:
The reasoning could be the same as for `LongListOffHeap` but we don't use it outside of the tests, so I can remove this constructor.

Reply:
Yes, this constructor is redundant. I'll remove it